src/stun.ts

```diff
@@ -115,67 +143,71 @@ export function getExternalIp(
       tids.splice(index, 1)
       const attr = res.getXorMappedAddressAttribute() ?? res.getMappedAddressAttribute()
 
+      console.log = backup
```

Comment:
The variable `backup` needs a better name.

Reply:
changed